pkg/sql/resolver.go

```diff
@@ -68,6 +68,34 @@ func (p *planner) ResolveMutableDatabaseDescriptor(
 	return desc.(*dbdesc.MutableDatabaseDescriptor), nil
 }
 
+// ResolveUncachedSchemaDescriptor looks up a schema from the store.
+func (p *planner) ResolveUncachedSchemaDescriptor(
```

Comment:
why do we need this method now?

Reply:
It's just used in a few schema changes where we need an uncached schema descriptor but don't need to update it. I meant for it to be parallel to `ResolveUncachedTableDescriptor`, etc.